data.json

```
@@ -192,5 +192,16 @@
       "url": "https://github.com/DesktopGooseUnofficial/ResourceHub/releases/download/honcker/Honcker.dll",
       "resourcehub": "https://desktopgooseunofficial.github.io/ResourceHub/mods/Honcker.html"
     }
+    "portalgoos": {
```

```suggestion
    },
    "portalgoos": {
```
This should be changed, or the launcher will crash for everyone due to incorrect JSON data.